Example/BitmovinConvivaAnalytics/ViewController.swift

```diff
@@ -82,23 +82,45 @@ class ViewController: UIViewController {
 
     var playerConfiguration: PlayerConfiguration {
         let playerConfiguration = PlayerConfiguration()
-        playerConfiguration.sourceItem = sourceItem
+        playerConfiguration.sourceItem = vodSourceItem
+//        playerConfiguration.sourceItem = vodSourceItemStartOffset
```

Comment:
Would it make sense to maybe provide a UI to switch between those sources? 🤔 like a TableView or any other select option to choose? 🤔 (definitely a NIT and not really necessary just thinking about a convenient way to test both use-cases without the need to re-building the app)

Reply:
Good input but keeping it as it is for now.